tests/main/snap-auto-import-asserts-spools/task.yaml

```diff
@@ -17,7 +17,7 @@ prepare: |
     #shellcheck source=tests/lib/ramdisk.sh
     . "$TESTSLIB/ramdisk.sh"
     setup_ramdisk
-    mkfs.vfat /dev/ram0
+    mkfs.ext3 /dev/ram0
```

Comment:
What is the motivation behind this change?

Reply:
explaned below :)